packages/story-editor/src/components/panels/document/taxonomies/FlatTermSelector.js

```diff
@@ -113,6 +128,19 @@ function FlatTermSelector({ taxonomy, canCreateTerms }) {
     [taxonomy, termCache]
   );
 
+  useEffect(() => {
+    (async function () {
+      const results = await addSearchResultsToCache(taxonomy, {
+        context: 'edit',
+        orderby: 'count',
+        order: 'desc',
+        hide_empty: true,
+      });
+      setMostUsed(results);
+    })();
```

Comment:
Was seeing these before but now not seeing any. Seems like my terms are coming back as an empty array. When is the `mosUsed` section populated?

Reply:
Yea, I think this is basically running on mount of the individual component because 
`taxonomy, addSearchResultsToCache` essentially never update in the lifetime of this component.

The only reason I didn't hoist this state to be held up at the provider level is because it would make the state keeping more complex to hold it for every flat term selector vs holding it in the components themselves. This set up equates to an extra network call every time the FlatTermSelector mounts but I think that's negligible

Reply:
Figured it out with max! Need to publish at least one story with tags for this to show up.